### PR TITLE
[EN] Review of the Article Style Guide

### PR DESCRIPTION
**Pull request by tom-arrow**
I just reviewed the ASG because I was going to translated it into french and saw some mistakes.

### CODE REVIEW

File: wiki/Article_Style_Guide/en.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 
-This Article Style Guide applies to all articles in English and other languages that does not have one yet!
+This Article Style Guide applies to all articles in English and other languages that do not have one yet!
 
 The Article Style Guide serves as a way to help keep consistency in clarity, formatting, and layout between all articles of the osu!wiki.
 
 All English articles should use plain English.
 Your word choice should explain the topic that the article aims to explain thoroughly in layman's terms.
-In other words, you should ask yourself, "If the reader is reading this, would they have to lookup any of the words?"
+In other words, you should ask yourself, "If the reader is reading this, would they have to look up any of the words?"
```
Comment (TPGPL) [inline wiki/Article_Style_Guide/en.md]: If the reader **was** reading this

File: wiki/Article_Style_Guide/en.md [hunk 2]
```diff
@@ -1,12 +1,12 @@
 # Article Style Guide
 
-This Article Style Guide applies to all articles in English and other languages that does not have one yet!
+This Article Style Guide applies to all articles in English and other languages that do not have one yet!
```
Comment (TPGPL) [inline wiki/Article_Style_Guide/en.md]: that -> which?

File: wiki/Article_Style_Guide/en.md [hunk 3]
```diff
@@ -17,7 +17,7 @@ A good osu!wiki writer/editor should read these reviews to help improve the over
 GFM (GitHub Flavored Markdown) is the supported markup language used in the osu!wiki.
 Usage of HTML is **heavily** discouraged!
 
-GFM has a small amount of markup that **anyone** can learn it in about 3 mintues!
+GFM has a small amount of markup that **anyone** can learn it in about 3 minutes!
```
Comment (TPGPL) [inline wiki/Article_Style_Guide/en.md]: can learn in about 3 minutes!

File: wiki/Article_Style_Guide/en.md [hunk 4]
```diff
@@ -68,7 +68,7 @@ If your language isn't listed below, please open an issue in the `osu-web` repos
 - `it.md`: Italian
 - `ja.md`: Japanese
 - `ko.md`: Korean
-- `nl.md`: Nederlands
+- `nl.md`: Dutch
 - `no.md`: Norwegian
 - `pl.md`: Polish
 - `pt.md`: Português
```
Comment (TPGPL) [inline wiki/Article_Style_Guide/en.md]: Portuguese?

**Comment by tom-arrow** [comment]
The thing is, I'm not a native speaker and there are probably other mistakes like the ones mentioned by @TPGPL . So it would be great if a native English speaker could review the whole article.

File: wiki/Article_Style_Guide/en.md (hunk 1 above)
Comment (oralekin) [inline wiki/Article_Style_Guide/en.md]: `If someone read this`?

Comment (TPGPL) [inline wiki/Article_Style_Guide/en.md]: this is second conditional i think, but yes, something like that

File: wiki/Article_Style_Guide/en.md [hunk 5]
```diff
@@ -68,11 +68,11 @@ If your language isn't listed below, please open an issue in the `osu-web` repos
...
 - `ja.md`: Japanese
 - `ko.md`: Korean
-- `nl.md`: Nederlands
+- `nl.md`: Dutch
 - `no.md`: Norwegian
 - `pl.md`: Polish
-- `pt.md`: Português
-- `pt-br.md`: Português (Brasil)
+- `pt.md`: Portuguese
+- `pt-br.md`: Portuguese (Brasil)
```
Comment (Kuron-kun) [inline wiki/Article_Style_Guide/en.md]: `Brazilian Portuguese` or `Portuguese (Brazil)`, but Brazilian Portuguese is prefered.

File: wiki/Article_Style_Guide/en.md [hunk 6]
```diff
@@ -1,12 +1,12 @@
...
 
-This Article Style Guide applies to all articles in English and other languages that does not have one yet!
+This Article Style Guide applies to all articles in English and other languages which do not have one yet!
 
 The Article Style Guide serves as a way to help keep consistency in clarity, formatting, and layout between all articles of the osu!wiki.
 
 All English articles should use plain English.
 Your word choice should explain the topic that the article aims to explain thoroughly in layman's terms.
-In other words, you should ask yourself, "If the reader is reading this, would they have to lookup any of the words?"
+In other words, you should ask yourself, "If someone was reading this, would they have to look up any of the words?"
```
Comment (kj415j45) [inline wiki/Article_Style_Guide/en.md]: > If someone read this

File: wiki/Article_Style_Guide/en.md [hunk 7]
```diff
@@ -17,7 +17,7 @@ A good osu!wiki writer/editor should read these reviews to help improve the over
 GFM (GitHub Flavored Markdown) is the supported markup language used in the osu!wiki.
 Usage of HTML is **heavily** discouraged!
 
-GFM has a small amount of markup that **anyone** can learn it in about 3 mintues!
+GFM has a small amount of markup that **anyone** can learn in about 3 minutes!
```
Comment (kj415j45) [inline wiki/Article_Style_Guide/en.md]: > can got it in about 3 minutes
?

File: wiki/Article_Style_Guide/en.md (hunk 6 above)
Comment (tom-arrow) [inline wiki/Article_Style_Guide/en.md]: > If someone read this, will they have to look up any of the words?

or

> If someone was reading this, would they have to lookup any of the words?

Idk maybe we should rephrase this

File: wiki/Article_Style_Guide/en.md (hunk 7 above)
Comment (tom-arrow) [inline wiki/Article_Style_Guide/en.md]: I think it's fine as it is

File: wiki/Article_Style_Guide/en.md [hunk 8]
```diff
@@ -68,11 +68,11 @@ If your language isn't listed below, please open an issue in the `osu-web` repos
...
+- `nl.md`: Dutch
 - `no.md`: Norwegian
 - `pl.md`: Polish
-- `pt.md`: Português
-- `pt-br.md`: Português (Brasil)
+- `pt.md`: Portuguese
+- `pt-br.md`: Brazilian Portuguese
 - `ru.md`: Russian
 - `th.md`: Thai
 - `tl.md`: Tagalog
```
Comment (oralekin) [inline wiki/Article_Style_Guide/en.md]: Change places of Taiwan Chinese and Traditional Chinese.

File: wiki/Article_Style_Guide/en.md (hunk 6 above)
Comment (kj415j45) [inline wiki/Article_Style_Guide/en.md]: I prefer the first one
Your article is not in the past, is in the future

File: wiki/Article_Style_Guide/en.md (hunk 8 above)
Comment (kj415j45) [inline wiki/Article_Style_Guide/en.md]: I mention that in my last review(L80 <-> L81)

**Comment by kj415j45** [comment]
about 32af998:
osu-wiki use osu-web support language:
osu-web use `Português (Brasil)` for `pt-br`

**Comment by kj415j45** [comment]
also 	4b55358:
image

File: wiki/Article_Style_Guide/en.md [hunk 9]
```diff
@@ -267,7 +267,7 @@ For example:
 
 ```
 The NC (Nightcore) mod is similar to the DT (Double Time) mod.
-While NC and DT increases the speed of the music by 25%, NC will change the pitch of the music and adds a clap and finish to each beat.
+While NC and DT increase the speed of the music by 25%, NC will change the pitch of the music and adds a clap and finish to each beat.
```
Comment (Deimosu) [inline wiki/Article_Style_Guide/en.md]: it increases by 50%, not 25%

File: wiki/Article_Style_Guide/en.md [hunk 10]
```diff
@@ -326,7 +326,7 @@ Note: the possessive forms of words (e.g. `the player's` or `the skinner's`) is
 
 ## Point of View
 
-When refering to the player, use `the player` or `a player`.
+When referring to the player, use `the player` or `a player`.
 
 **Never** use `he`/`his` or `she`/`hers`!
```
Comment (TPGPL) [inline wiki/Article_Style_Guide/en.md]: she/her?
